### PR TITLE
Allow multiple limiters for multiple fields.

### PR DESCRIPTION
**Pull request by zitkat**
Preliminary implementation to start discussion. Main issue I see is mapping between fields, variables, limiters and slices of solution vector as occuring in time stepping solvers.

### CODE REVIEW

**Comment by rc** [comment]
Thanks @zitkat!  There seems to be a trailing white-space in the new example file. Do we need to add it? It would require docstring update and some explanation, as it is rather contrived (two independent equations).

File: sfepy/solvers/ts_dg_solvers.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import numpy.linalg as nla
 
 # sfepy imports
+from limiters import ComposedLimiter, IdentityLimiter
```
[the remark this answers is not in the document]
Comment (zitkat) [reply sfepy/solvers/ts_dg_solvers.py]: Fixed it.

**Comment by rc** [comment]
Thanks @zitkat, the tests pass now. See also my previous comment - the example either needs some polishing, or simply omit it from the PR for the moment.

**Comment by zitkat** [comment]
> Thanks @zitkat, the tests pass now. See also my previous comment - the example either needs some polishing, or simply omit it from the PR for the moment.

No problem, I removed the example, we can add better later.

**Comment by rc** [comment]
Thanks @zitkat! Do you want to update the DOF extraction properly in this PR, or we merge now, and do a follow-up PR?

**Comment by rc** [comment]
Well, let's merge it so that the PR does not hang here indefinitely... Thanks!

**Pull request merged**